exercise_0/main.cpp

```diff
@@ -0,0 +1,80 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+
+struct Result {
+    int vowels;
+    int consonant;
+    int others;
+};
+
+Result countLetter(std::string firstLetter) {
+    Result result;
+    std::string vowels = "AEIOUYaeouy";
+    int countVowels = 0;
```

Comment:
You can use the struct fields directly and skip local variables, i.e. `++result.vowels;`